spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:example) { get users_path }
+    it 'Shoud return a 200 status' do
+      expect(response).to have_http_status(200)
+    end
+
+    it 'Should show the index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'Should incldue the correct HTML' do
+      expect(response.body).to include '<div class="user_div">'
+    end
+  end
+
+  describe 'GET /show' do
+    before(:example) { get '/users/12' }
+    it 'Should retun a 200 status' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'Should show the index template' do
+      expect(response).to render_template :show
+    end
+
+    it 'Should incldue the correct HTML' do
+      expect(response.body).to include '<div class="user_detail_div">'
+    end
```

Comment:
- [ ] The following tests for the `show` action of the User are not passing. Please see the image attached below and try to make all the tests green ✔️ 
![image](https://user-images.githubusercontent.com/46839548/204407233-18bdfe84-ba3e-4d29-bdf1-98b15228aba8.png)